src/index.js

```diff
@@ -1,14 +1,25 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import { configureStore } from "@reduxjs/toolkit";
+import { Provider } from "react-redux";
+import todoReducer from "./features/todo";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const store = configureStore({
+    reducer: {
+        todo: todoReducer,
+    },
+});
```

Comment:
For good practice, the store has to be outside of this index file maybe you can create a folder called `store` and that folder will have a file called `index.js` with the store configuration.